gpMgmt/bin/generate-greenplum-path.sh

```diff
@@ -27,13 +27,20 @@ else
 fi
 EOF
 
+# only override PATH when Python used during ./configure isn't included in.
+if [[ $(python3 -c "import sys; print(sys.executable)") == "${WHICHPYTHON}" ]]; then
+  PATHSTR='${GPHOME}/bin:${PATH}'
+else
+  PATHSTR='${GPHOME}/bin:${PYTHONBINDIR}:${PATH}'
+fi
+
 cat <<EOF
 PYTHONBINDIR="$(dirname "${WHICHPYTHON}")"
+PATH="${PATHSTR}"
 EOF
 
 cat <<"EOF"
 PYTHONPATH="${GPHOME}/lib/python"
-PATH="${GPHOME}/bin:${PYTHONBINDIR}:${PATH}"
```

Comment:
can put $PATH in front ? like `PATH="${PATH}:${GPHOME}/bin:${PYTHONBINDIR}"`

Reply:
Probably no. The system `psql` like commands(comes from the standard postgres) will have higher precedence over the commands in cbdb.